avaframe/out3Plot/outQuickPlot.py

```diff
@@ -206,7 +208,8 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
                  (centiles[0], unit, centiles[1], unit),
                  horizontalalignment='left', verticalalignment='bottom', transform=ax4.transAxes)
 
-    fig.savefig(os.path.join(outDir, 'Diff_%s_%s.%s' % (avaName, simName, pU.outputFormat)))
+    saveName1 = outDir / ('Diff_%s_%s.%s' % (avaName, simName, pU.outputFormat))
```

Comment:
Minor comment: names like saveNameDiff and saveNameProfile would be easier to read instead of xxx1 and xxx2...

Reply:
adjusted